tests/v8_help.out

```diff
@@ -0,0 +1 @@
+[WILDCARD]
```

Comment:
Is this actually testing anything ? It seems empty output would satisfy it.

Reply:
I wanted to test that it actually outputs something, matching some initial line would be better but there are some strange flags on first line of V8's help:
```
SSE3=1 SSSE3=1 SSE4_1=1 SAHF=1 AVX=1 FMA3=1 BMI1=1 BMI2=1 LZCNT=1 POPCNT=1 ATOM=0
```
I think they can change, so makes no sense to match against them.

Reply:
Those change. But probably some other parts of the output are stable. You could start and end with a wildcard.